shared/github_oauth.go

```diff
@@ -100,3 +115,51 @@ func NewGitHubOAuth(ctx context.Context) (GitHubOAuth, error) {
 
 	return &githubOAuthImp{ctx: ctx, conf: oauth, ds: store}, nil
 }
+
+var secureCookie *securecookie.SecureCookie
```

Comment:
I know this is just moving code we already had, but do you happen to know why we store `secureCookie` in a global, rather than create it anew each time `GetSecureCookie` is called? Do we have reason to expect it to be a slow operation?

Reply:
Good question. I think the original purpose is to avoid the datastore lookup repeatedly from `GetSecret` methods but turns out that they are memcached after the initial lookup. Having a global secureCookie will not speed things up.